src/core.js

```diff
@@ -91,6 +91,8 @@ const patchFactory = function(run) {
     doc = node.ownerDocument;
     currentParent = node.parentNode;
 
+    importNode(node);
```

Comment:
Can't do that because Incremental DOM needs to be robust against DOM creation performed outside of it.